config/src/defaults.rs

```diff
@@ -127,6 +127,18 @@ pub trait Defaults {
         5
     }
 
+    /// Default initial block reward
+    fn consensus_constants_initial_block_reward(&self) -> u64 {
+        // 250 wits
+        250 * 1_000_000_000
+    }
+
+    /// Default halving period
+    fn consensus_constants_halving_period(&self) -> u32 {
+        // 3.5M epochs * (45 secs/epoch) ~> 5 years
+        3_500_000
```

Comment:
This will be the first timestamp with halved reward:

1760166000 // 3500000 * 45 + 1602666000
10/11/2025 @ 7:00am (UTC)